rapidoid/content.md

```diff
@@ -0,0 +1,135 @@
+# What is Rapidoid?
+
+Rapidoid is an extremely fast HTTP server and modern Java web framework / application container, with a strong focus on high productivity and high performance.
+
+> [www.rapidoid.org](http://www.rapidoid.org)
+
+%%LOGO%%
+
+# How to use this image
+
+To quickly start Rapidoid and display some basic usage help, run:
+
+```console
+$ docker run --rm %%REPO%% --help
+```
+
+Rapidoid can be used in different ways:
+
+-	as a web tool, to quickly prototype RESTful web services from the command line
+-	as a HTTP server, to serve static resources
+-	as a Java web application framework/container, to deploy a web application JAR
+
+## Quickly prototyping SQL-powered RESTful web services
+
+To quickly prototype SQL-powered RESTful web services from the command line, you need to link the database container to the Rapidoid container. The MySQL containers should be linked under name `mysql`, and PostgreSQL containers under name `postgres`.
+
+This example starts a new MySQL container and links it under name `mysql` in the Rapidoid container, where a RESTful service is defined by specifying SQL query for the route `GET /users`. The service returns the result (a list of MySQL users) in JSON format.
+
+```console
+DB_ID=$(docker run -d -e MYSQL_ROOT_PASSWORD=db-pass mysql)
```

Comment:
I apologize for my late reply, I just got back from vacation. I will take care of this today...